node/src/BaseClient.ts

```diff
@@ -251,7 +251,7 @@ export type ReturnType =
     | ReturnTypeAttribute
     | ReturnType[];
 
-export type GlideString = string | Uint8Array;
+export type GlideString = string | Buffer;
```

Comment:
Add `GlideString` to exports in `node/npm/glide/index.ts`.
Please also add a doc for this type.